### ISSUE TITLE
Type 'en' is not a subtype of type 'S'

### ISSUE THREAD

**Issue by phamanhminh1208** [long1eu/flutter_i18n#20]
I use flutter sdk 0.9.4 and the default config as in your plugin document:
```
Widget build(BuildContext context) {
    return new MaterialApp(
      onGenerateTitle: (BuildContext context) => S.of(context).app_title,
      localizationsDelegates: [S.delegate,],
      supportedLocales: S.delegate.supportedLocales,
      localeResolutionCallback:
          S.delegate.resolution(fallback: new Locale("en", "")),
      theme: new ThemeData(
                primarySwatch: Colors.blue,
      ),
      home: new HomeView(),
    );
}
```

And in HomeView, I use like that:
`new Text(S.of(context).start),`

Then it makes error like below:

```
22:33:27.416 2 info flutter.tools I/flutter (11541): ══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
22:33:27.416 3 info flutter.tools I/flutter (11541): The following assertion was thrown building HomeView(dirty):
22:33:27.416 4 info flutter.tools I/flutter (11541): type 'en' is not a subtype of type 'S'
22:33:27.416 5 info flutter.tools I/flutter (11541): 
22:33:27.416 6 info flutter.tools I/flutter (11541): Either the assertion indicates an error in the framework itself, or we should provide substantially
22:33:27.416 7 info flutter.tools I/flutter (11541): more information in this error message to help you determine and fix the underlying cause.
22:33:27.416 8 info flutter.tools I/flutter (11541): In either case, please report this assertion by filing a bug on GitHub:
22:33:27.416 9 info flutter.tools I/flutter (11541):   https://github.com/flutter/flutter/issues/new
22:33:27.416 10 info flutter.tools I/flutter (11541): 
22:33:27.416 11 info flutter.tools I/flutter (11541): When the exception was thrown, this was the stack:
22:33:27.432 12 info flutter.tools I/flutter (11541): #0      _LocalizationsState.resourcesFor (package:flutter/src/widgets/localizations.dart:533:13)
22:33:27.432 13 info flutter.tools I/flutter (11541): #1      Localizations.of (package:flutter/src/widgets/localizations.dart:445:39)
22:33:27.447 14 info flutter.tools I/flutter (11541): #2      S.of (package:eye_test_game/generated/i18n.dart:19:21)
22:33:27.447 15 info flutter.tools I/flutter (11541): #3      HomeView.build (package:eye_test_game/presentations/home/home_view.dart:23:33)
22:33:27.447 16 info flutter.tools I/flutter (11541): #4      StatelessElement.build (package:flutter/src/widgets/framework.dart:3731:28)
22:33:27.447 17 info flutter.tools I/flutter (11541): #5      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3678:15)
22:33:27.447 18 info flutter.tools I/flutter (11541): #6      Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.447 19 info flutter.tools I/flutter (11541): #7      StatelessElement.update (package:flutter/src/widgets/framework.dart:3738:5)
22:33:27.447 20 info flutter.tools I/flutter (11541): #8      Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 21 info flutter.tools I/flutter (11541): #9      SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.447 22 info flutter.tools I/flutter (11541): #10     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 23 info flutter.tools I/flutter (11541): #11     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.447 24 info flutter.tools I/flutter (11541): #12     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.447 25 info flutter.tools I/flutter (11541): #13     StatelessElement.update (package:flutter/src/widgets/framework.dart:3738:5)
22:33:27.447 26 info flutter.tools I/flutter (11541): #14     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 27 info flutter.tools I/flutter (11541): #15     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.447 28 info flutter.tools I/flutter (11541): #16     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 29 info flutter.tools I/flutter (11541): #17     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.447 30 info flutter.tools I/flutter (11541): #18     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 31 info flutter.tools I/flutter (11541): #19     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.447 32 info flutter.tools I/flutter (11541): #20     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 33 info flutter.tools I/flutter (11541): #21     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.447 34 info flutter.tools I/flutter (11541): #22     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 35 info flutter.tools I/flutter (11541): #23     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.447 36 info flutter.tools I/flutter (11541): #24     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.447 37 info flutter.tools I/flutter (11541): #25     StatefulElement.update (package:flutter/src/widgets/framework.dart:3835:5)
22:33:27.447 38 info flutter.tools I/flutter (11541): #26     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 39 info flutter.tools I/flutter (11541): #27     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.447 40 info flutter.tools I/flutter (11541): #28     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.447 41 info flutter.tools I/flutter (11541): #29     StatelessElement.update (package:flutter/src/widgets/framework.dart:3738:5)
22:33:27.447 42 info flutter.tools I/flutter (11541): #30     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 43 info flutter.tools I/flutter (11541): #31     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.447 44 info flutter.tools I/flutter (11541): #32     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.447 45 info flutter.tools I/flutter (11541): #33     StatefulElement.update (package:flutter/src/widgets/framework.dart:3835:5)
22:33:27.447 46 info flutter.tools I/flutter (11541): #34     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 47 info flutter.tools I/flutter (11541): #35     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.447 48 info flutter.tools I/flutter (11541): #36     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 49 info flutter.tools I/flutter (11541): #37     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.447 50 info flutter.tools I/flutter (11541): #38     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.447 51 info flutter.tools I/flutter (11541): #39     ProxyElement.update (package:flutter/src/widgets/framework.dart:3947:5)
22:33:27.447 52 info flutter.tools I/flutter (11541): #40     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 53 info flutter.tools I/flutter (11541): #41     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.447 54 info flutter.tools I/flutter (11541): #42     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.447 55 info flutter.tools I/flutter (11541): #43     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.447 56 info flutter.tools I/flutter (11541): #44     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.447 57 info flutter.tools I/flutter (11541): #45     StatefulElement.update (package:flutter/src/widgets/framework.dart:3835:5)
22:33:27.447 58 info flutter.tools I/flutter (11541): #46     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.463 59 info flutter.tools I/flutter (11541): #47     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.463 60 info flutter.tools I/flutter (11541): #48     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.463 61 info flutter.tools I/flutter (11541): #49     StatelessElement.update (package:flutter/src/widgets/framework.dart:3738:5)
22:33:27.463 62 info flutter.tools I/flutter (11541): #50     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.463 63 info flutter.tools I/flutter (11541): #51     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4803:14)
22:33:27.463 64 info flutter.tools I/flutter (11541): #52     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.463 65 info flutter.tools I/flutter (11541): #53     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.463 66 info flutter.tools I/flutter (11541): #54     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.463 67 info flutter.tools I/flutter (11541): #55     ProxyElement.update (package:flutter/src/widgets/framework.dart:3947:5)
22:33:27.463 68 info flutter.tools I/flutter (11541): #56     Element.updateChild (package:flutter/src/widgets/framework.dart:2729:15)
22:33:27.463 69 info flutter.tools I/flutter (11541): #57     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
22:33:27.463 70 info flutter.tools I/flutter (11541): #58     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
22:33:27.463 71 info flutter.tools I/flutter (11541): #59     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2273:33)
22:33:27.463 72 info flutter.tools I/flutter (11541): #60     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:673:20)
22:33:27.463 73 info flutter.tools I/flutter (11541): #61     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:208:5)
22:33:27.463 74 info flutter.tools I/flutter (11541): #62     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:990:15)
22:33:27.463 75 info flutter.tools I/flutter (11541): #63     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:930:9)
22:33:27.463 76 info flutter.tools I/flutter (11541): #64     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.scheduleWarmUpFrame.<anonymous closure> (package:flutter/src/scheduler/binding.dart:751:7)
22:33:27.463 77 info flutter.tools I/flutter (11541): #66     _Timer._runTimers (dart:isolate/runtime/libtimer_impl.dart:382:19)
22:33:27.463 78 info flutter.tools I/flutter (11541): #67     _Timer._handleMessage (dart:isolate/runtime/libtimer_impl.dart:416:5)
22:33:27.463 79 info flutter.tools I/flutter (11541): #68     _RawReceivePortImpl._handleMessage (dart:isolate/runtime/libisolate_patch.dart:171:12)
22:33:27.463 80 info flutter.tools I/flutter (11541): (elided one frame from package dart:async)
22:33:27.463 81 info flutter.tools I/flutter (11541): ════════════════════════════════════════════════════════════════════════════════════════════════════
22:33:27.525 82 info flutter.tools Reloaded 3 of 431 libraries in 1,049ms.

```

**Comment by noordawod** [comment]
This should be fixed with #19. Waiting for JetBrains to approve the latest build: https://plugins.jetbrains.com/plugin/10128-flutter-i18n

**Comment by noordawod** [comment]
@long1eu this can be closed now I think.
